backend/src/aggregator.rs

```diff
@@ -1,29 +1,32 @@
-use std::collections::{HashMap, HashSet};
 use actix::prelude::*;
-use actix_web_actors::ws::{CloseReason, CloseCode};
+use actix_web_actors::ws::{CloseCode, CloseReason};
 use ctor::ctor;
+use std::collections::{HashMap, HashSet};
 
-use crate::node::connector::{Mute, NodeConnector};
-use crate::feed::connector::{FeedConnector, Connected, FeedId};
-use crate::util::DenseMap;
+use crate::chain::{self, Chain, ChainId, GetNodeNetworkState, Label};
+use crate::feed::connector::{Connected, FeedConnector, FeedId};
 use crate::feed::{self, FeedMessageSerializer};
-use crate::chain::{self, Chain, ChainId, Label, GetNodeNetworkState};
+use crate::node::connector::{Mute, NodeConnector};
 use crate::types::{ConnId, NodeDetails, NodeId};
+use crate::util::{DenseMap, Hash};
 
 pub struct Aggregator {
+    hashes: HashMap<Hash, ChainId>,
```

Comment:
```suggestion
    genesis_hashes: HashMap<Hash, ChainId>,
```

Reply:
Oof, I missed that when looking at code.